src/solver/variable/include/antares/solver/variable/surveyresults/reportbuilder.hxx

```diff
@@ -354,9 +356,11 @@ private:
 
                 logs.info() << "Exporting results : " << area.name << " :: " << cluster->name();
                 // The new output
-                results.data.output.clear();
-                results.data.output << results.data.originalOutput << SEP << "areas" << SEP
-                                    << area.id << SEP << "thermal" << SEP << cluster->id();
+                std::filesystem::path path = std::string(results.data.originalOutput);
+                path /= std::filesystem::path("areas") / area.id.to<std::string>() / "thermal"
+                        / cluster->id();
```

Comment:
If you have a `typedef`, you can just use
```suggestion
                path /= fs("areas") / area.id.to<std::string>() / "thermal"
                        / cluster->id();
```

Reply:
It's a hxx file, i don't use typedef